db/patterns/report_uri.eno

```diff
@@ -0,0 +1,11 @@
+name: Report URI
+category: essential
+website_url: https://report-uri.com
+organization: report_uri
+
+--- domains
+||report-uri.com^$3p
```

Comment:
This looks like a filter. I assume, it should be only `report-uri.com` here.

Reply:
My mistake